code/obj/item/device/radio.dm

```diff
@@ -658,6 +658,30 @@ var/list/headset_channel_lookup
 	item_state = "signaler"
 	desc = "A small beacon that is tracked by the Teleporter Computer, allowing things to be sent to its general location."
 	burn_possible = 0
+	anchored = 1
+
+	attackby(obj/item/I as obj, mob/user as mob)
+		if (isscrewingtool(I))
+			if (src.anchored)
+				playsound(src.loc, "sound/items/Screwdriver.ogg", 50, 1)
+				user.show_text("You start unscrewing [src] from the floor.", "blue")
+				if (do_after(user, 3 SECONDS))
```

Comment:
just make this instant imo. no reason to make people wait 3 seconds to unanchor a beacon

Reply:
otherwise good change